src/operator/nn/pooling.cc

```diff
@@ -377,8 +377,7 @@ We can see that Lp pooling stands between those two, in practice the most common
 
 For each window ``X``, the mathematical expression for Lp pooling is:
 
-..math::
-  f(X) = \sqrt{p}{\sum\limits_{x \in X} x^p}
+:math:`f(X) = \sqrt[p]{\sum_{x}^{X} x^p}`
```

Comment:
I am not an expert in this. Someone familiar with it may want to check the math here since this change is not mentioned in the PR description.

Reply:
This change is mentioned in the description in the `changes` section.

Reply:
Thanks